src/break/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import injectSheet from 'react-jss';
+import { setPropTypes, compose } from 'recompose';
+
+import {Section} from '../section';
+
+const styles = {
+  wrapper: {
+    background: ({gradientOne, gradientTwo}) => `linear-gradient(to top right, ${gradientOne}, ${gradientTwo})`,
+    textAlign: 'left',
+    display: 'flex',
+  },
+  content: {
+    padding: [48, 36],
+    color: ({fontColor}) => fontColor,
```

Comment:
I feel this is unnecessary. If you can provide children, you can specify the font colour directly on those and this prop will be overridden.